lib/zhr_devs/identity_management/events/logged_in.ex

```diff
@@ -0,0 +1,4 @@
+defmodule ZhrDevs.IdentityManagement.Events.LoggedIn do
+  @derive Jason.Encoder
```

Comment:
I assume that we can't do an AST here because Json.Encoder would happen before unfolding algae's defdata? Probably something to improve some time later, maybe integrate into `defdata`?! :thinking: Probably warrants an issue.

Reply:
I'll add it locally and it works with Algae.

Reply:
The other question is wether we **need** to use it for Commands, for example.